From 7970e9135c736bb61cf28ac564eebb5dea47667e Mon Sep 17 00:00:00 2001
From: Yonathan Yusim <yonathan@boxee.tv>
Date: Wed, 6 Jul 2011 18:34:58 +0300
Subject: [PATCH 09/14] add video/MP4V-ES

---
 djmount/media_file.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/djmount/media_file.c b/djmount/media_file.c
index 7baa81b..8ba99cc 100644
--- a/djmount/media_file.c
+++ b/djmount/media_file.c
@@ -92,6 +92,7 @@ static const MimeType MIMES[] = {
 	{ "video/x-matroska",			NULL,	"mkv"	},
 	{ "video/mpeg",				NULL,	"mpg"	},
 	{ "video/mp2p",				NULL,	"vob"	},
+	{ "video/MP4V-ES",			NULL,   "m4v"   },
 	// Default for all other video files : asf, mpeg2, x-ms-wmv, ...
 	{ "video/",				NULL,	NULL	},
 
@@ -162,7 +163,7 @@ MediaFile_GetPreferred (const DIDLObject* const o, MediaFile* file)
 		if (uri == NULL || protocol == NULL || 
 		    sscanf (protocol, "http-get:*:%63[^:;]", mimetype) != 1) 
 			continue; // ---------->
-			
+
 		const MimeType* format = MIMES;
 		while (format->mimetype != NULL && !found) {
 			if (strncmp (mimetype, format->mimetype, 
-- 
2.0.0.rc0

